Phaethon/Tests/Tests.csproj

```diff
@@ -264,7 +265,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="AcceptanceTests\" />
-    <Folder Include="UnitTests\" />
```

Comment:
![image](https://user-images.githubusercontent.com/22789107/49926867-dd9a8c00-febc-11e8-9aa9-cfbd47d204c4.png)
Why did you remove the Acceptance and Unit Tests folders?